test/unit/modules/signatures.js

```diff
@@ -14,66 +14,226 @@
+			library.modules.signatures.onBind(library.modules);
+			done();
+		});
+
 		describe('modules', () => {
-			it('should assign accounts');
+			it('should assign accounts', done => {
+				expect(privateModules).to.have.property(
+					'accounts',
+					library.modules.accounts
+				);
+				done();
+			});
 
-			it('should assign transactions');
+			it('should assign transactions', done => {
+				expect(privateModules).to.have.property(
+					'transactions',
+					library.modules.transactions
+				);
+				done();
+			});
 
-			it('should assign transport');
+			it('should assign transport', done => {
+				expect(privateModules).to.have.property(
+					'transport',
+					library.modules.transport
+				);
+				done();
+			});
 		});
 
 		describe('assetTypes', () => {
-			it('should call bind on signature logic with scope.accounts');
+			it('should call bind on signature logic with scope.accounts', done => {
+				expect(signatureLogicSpy).to.be.calledWith(library.modules.accounts);
+				done();
+			});
 		});
 	});
 
 	describe('shared', () => {
-		describe('postSignatures', () => {
-			describe('when modules not are loaded', () => {
-				it('should call callback with ApiError');
-
-				it(
-					'should call callback with ApiError containing message = "Blockchain is loading"'
+		describe('postSignature', () => {
+			const req = {
+				body: {
+					signature: 'aSignature',
+					transactionId: 'aTransactionId',
+					publicKey: 'aPublicKey',
+				},
+			};
+
+			let postSignatureSpy;
+
+			before(done => {
+				postSignatureSpy = sinonSandbox.spy(
+					library.modules.transport.shared,
+					'postSignature'
+				);
+				library.modules.signatures.shared.postSignature(
+					req.body,
+					(error, response) => {
+						done();
+					}
 				);
+			});
 
-				it('should call callback with ApiError containing code = 500');
+			it('should call modules.transport.shared.postSignature with req.body', done => {
+				expect(postSignatureSpy).to.be.calledWith({ signature: req.body });
+				sinonSandbox.restore();
+				done();
 			});
 
-			describe('when modules are loaded', () => {
-				it('should call modules.transport.shared.postSignatures with req.body');
+			describe('when modules.transport.shared.postSignature fails with result', () => {
+				let postSignatureStub;
+
+				before(done => {
+					postSignatureStub = sinonSandbox.stub(
+						library.modules.transport.shared,
+						'postSignature'
+					);
+					done();
+				});
+
+				after(done => {
+					sinonSandbox.restore();
+					done();
+				});
 
-				describe('when modules.transport.shared.postSignatures fails with result', () => {
-					it('should call callback with ApiError');
+				it('should call callback with ApiError', done => {
+					postSignatureStub.yields(null, { success: false });
 
-					it(
-						'should call callback with ApiError containing message = "Blockchain is loading"'
+					library.modules.signatures.shared.postSignature(
+						req.body,
+						(error, result) => {
```

Comment:
you don't have to define the result argument since it's the second argument and not necessary here.

Reply:
Done